portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+var usingBlobForm = false;
```

Comment:
Can you add some documentation explaining what this is used for? It seems to me like you don't need it if useBlobForm is only called once?

Reply:
This is a vestigial leftover from an implementation I tried locally, I will remove it